File: hw/santorum/u1/HW11ChickenCoops/ChikenFarmSimulatorPython/model/chickenCoop.py
from model.chicken import Chicken

class ChickenCoop:
    def __init__(self, coopId):
        self.coopId = coopId
        self.chickens = [] 

    def addChicken(self, chicken: Chicken):
        self.chickens.append(chicken)

    def __str__(self):
        if not self.chickens:
            return f"Coop {self.coopId}: Empty"
        chickensStr = "\n  ".join([str(c) for c in self.chickens])
        return f"Coop {self.coopId}:\n  {chickensStr}"

File: hw/santorum/u1/HW11ChickenCoops/ChikenFarmSimulatorJavaScript/model/chickenCoop.js
import { Chicken } from './chicken.js';

export class ChickenCoop {
    constructor(coopId) {
        this.coopId = coopId;
        this.chickens = []; // array como ArrayList
    }

    addChicken(chicken) {
        this.chickens.push(chicken);
    }

    toString() {
        if (this.chickens.length === 0) return `Coop ${this.coopId}: Empty`;
        const chickensStr = this.chickens.map(c => c.toString()).join('\n  ');
        return `Coop ${this.coopId}:\n  ${chickensStr}`;
    }
}
